test/UnitTest/System.Web.OData.Test/packages.config

```diff
@@ -6,9 +6,9 @@
   <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.2" targetFramework="net45" />
   <package id="Microsoft.Extensions.DependencyInjection" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.OData.Core" version="7.3.1" targetFramework="net45" />
-  <package id="Microsoft.OData.Edm" version="7.3.1" targetFramework="net45" />
-  <package id="Microsoft.Spatial" version="7.3.1" targetFramework="net45" />
+  <package id="Microsoft.OData.Core" version="7.5.0-Nightly201802011100" targetFramework="net45" />
```

Comment:
Is there any code referring to the Nightly builds?

Reply:
Your observation is correct. The version is set to the pre-release 7.4.1 (nightly build version), but the WebAPI test hit an issue in 7.4.1 release. Therefore we need to eventually update it to 7.4.3+.

---
In reply to: [176836894](https://github.com/OData/WebApi/pull/1225#discussion_r176836894) [](ancestors = 176836894)